cypress/support/step_definition/dashboard/dashboard.spec.js

```diff
@@ -1,10 +1,12 @@
 import { Given, When, Then } from 'cypress-cucumber-preprocessor/steps'
+import Page_Navigation from '../../pages/common/pageNavigation'
 import Dashboard from '../../pages/dashboard/pageDashboard'
 
+const selectNavigation = new Page_Navigation()
```

Comment:
also object name here should be the same as class name --> const pageNavigation = new Page_Navigation()

Reply:
done